rca/static_src/javascript/programmes/components/ProgrammesResults/ProgrammeTeaser.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { programmePageShape } from '../../programmes.types';
+import {
+    SHORT_COURSE_PAGE_TYPE,
+    programmePageShape,
+} from '../../programmes.types';
 
 /**
- * A programme’s teaser info, to be displayed as part of search results.
+ * A programme or short course’s teaser info, to be displayed as part of search results.
+ * "Degree level" and "Pathway blocks" are for programmes only. Other fields are shared.
  */
 const ProgrammeTeaser = ({ programme, onMouseOver, onFocus }) => {
-    const {
-        meta,
-        title,
-        degree_level,
-        programme_description_subtitle,
-        pathway_blocks,
-    } = programme;
+    const { meta, title, summary, degree_level, pathway_blocks } = programme;
+    const isShortCourse = meta.type === SHORT_COURSE_PAGE_TYPE;
+    // Short courses have no degree level. Display a distinguishing label instead.
+    const degreeLabel = isShortCourse ? 'Short course' : degree_level.title;
```

Comment:
Note this is the label that might need changing based on feedback on the ticket.